client/components/frontpage/ChinguSection.js

```diff
@@ -0,0 +1,223 @@
+import React from "react";
+import Link from "next/link";
+import styled, { css } from "styled-components";
+import { breakpoint } from "../../frontend-config";
+import { Heading2, TextBody } from "../shared/styles";
+
+const SVG = () => {
+  return (
+    <svg
```

Comment:
This SVG should probably be replaced with "home-chingu.svg" - I see that SVG is added, but I don't see any references 🙂

Reply:
@Soreg Thank you for the feedback. I have made changes to import SVG file directly.